geode-dunit/src/main/java/org/apache/geode/internal/cache/functions/TestFunction.java

```diff
@@ -1011,6 +1018,20 @@ private void executeWithLastResult(FunctionContext context) {
     context.getResultSender().lastResult(context.getArguments());
   }
 
+  private void executeParent(FunctionContext context) {
+    try {
```

Comment:
Just curious what this Thread.sleep() is used for?  Might justify a comment.

Reply:
That was an attempt to get all the parent functions executing before the child functions were invoked. Its definitely hacky and not actually necessary. I took it out.